strings/src/main/res/values/strings.xml

```diff
@@ -990,4 +990,6 @@
     <string name="not_a_directory">%s exists, but is not a directory</string>
 
     <string name="select_odk_shortcut">Select ODK Shortcut</string>
+    <string name="record_audio">Record audio</string>
+    <string name="stop_recording_confirmation">Do you really want to stop recording?</string>
```

Comment:
This one can move with the string above. 

Maybe add more details with like "This form requests background audio recording. Disabling it will stop recording and discard existing audio. Are you sure you want to proceed?"